File: programmers/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;
    while(true)
    {
        answer.push_back(n%10);
        n = n/10;
        if(n == 0)
        {
            break;
        }
    }
    return answer;
}

File: programmers/level1/자연수뒤집어배열로만들기.java
class Solution {
  public int[] solution(long n) {
      String str = n+"";
      int len = str.length();
      int[] answer = new int[len];
      
      for(int i=0; i<len; i++) {
          answer[i] = (int)(n%10);
          n=n/10;
      }
      
      return answer;
  }
}